webui/react/src/pages/TrialLogs.tsx

```diff
@@ -70,9 +77,28 @@ const TrialLogs: React.FC = () => {
         publicSubject: 'Download Failed',
         type: ErrorType.Ui,
       });
-    });
+    }
+
+    setIsDownloading(false);
   }, [ id ]);
 
+  const handleDownloadLogs = useCallback(() => {
+    Modal.confirm({
+      content: <div>
+        For large trial logs it will take a significant amount of
+        time and browser memory to complete download. We recommend
+        using the CLI to download instead:
+        <code className="block">
+          det trial logs {id}
```

Comment:
suggestion: Let's redirect to a file whose name mimics the behavior we'd get with the download button.
e.g., `det trial logs {id} > {experiment_id}_{trial_id}_logs.txt`